tools/scripts/authz/dao_genesis_authorizations.json

```diff
@@ -188,6 +188,15 @@
     },
     "expiration": "2500-01-01T00:00:00Z"
   },
+  {
+    "granter": "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t",
+    "grantee": "pokt1eeeksh2tvkh7wzmfrljnhw4wrhs55lcuvmekkw",
+    "authorization": {
+      "@type": "\/cosmos.authz.v1beta1.GenericAuthorization",
+      "msg": "\/pocket.migration.MsgUpdateParams"
```

Comment:
If we add another param here, need to make sure to implement `MsgUpdateParam`.

Non-actionable.

Reply:
Indeed. I intentionally chose to skip it for now.